app/templates/components/user-count.hbs

```diff
@@ -1,10 +1,10 @@
-<h1>user count: {{ getUser.length}}</h1>
+{{!--<h1>user count: {{ getUser.length}}</h1>--}}
 
 <div class="avatar-wrapper">
 {{#each getUser as |user|}}
 
-{{#user-avatar user=user}}
-{{/user-avatar}}
+{{#user-card user=user}}
+{{/user-card}}
```

Comment:
Use angle brackets in the hbs file.

`<UserCard @user={{this.user}} />`

Reply:
I have fixed this change.